### PR TITLE
Add web components to public website

### PR DESCRIPTION
**Pull request by micahgodbolt**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ yarn change`

#### Description of changes

Giving WC a presence on the existing public website. PR still needs clarity before merge:

1. `webcomponents` as url shorthand
2. urls to docs/DNA
3. Logos, colors and other style details

#### Focus areas to test

(optional)

### CODE REVIEW

File: apps/public-docsite/src/SiteDefinition/SiteDefinition.pages/Controls/index.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 export * from './Controls.pages';
 export * from './web';
+export * from './wc';
```
[the remark this answers is not in the document]
Comment (ecraig12345) [reply apps/public-docsite/src/SiteDefinition/SiteDefinition.pages/Controls/index.ts:3]: @micahgodbolt ping on this comment, or if you'd rather leave it as-is it's not super important

File: packages/react-docsite-components/src/components/PlatformPicker/PlatformPicker.tsx [hunk 2]
```diff
@@ -33,7 +33,7 @@ export class PlatformPicker<TPlatforms extends string = string> extends React.Pu
                     href={pages && this._getFirstPageUrl(pages)}
                     className={css(styles.button, platformKey === activePlatform && styles.activePlatform)}
                     iconProps={{
-                      iconName: icon,
+                      iconName: platform.name !== 'Web Components' ? icon : '',
```
Comment (ecraig12345) [inline packages/react-docsite-components/src/components/PlatformPicker/PlatformPicker.tsx:36]: Can you add a comment about why web components doesn't have an icon here?

File: apps/public-docsite/src/SiteDefinition/SiteDefinition.pages/Controls/index.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (micahgodbolt) [reply apps/public-docsite/src/SiteDefinition/SiteDefinition.pages/Controls/index.ts:3]: @ecraig12345 yeah, i considered changing file/folder names as well. It didn't have any impact on the public facing site though, so I didn't prioritize the change.

File: packages/react-docsite-components/src/components/PlatformPicker/PlatformPicker.tsx (hunk 2 above)
Comment (micahgodbolt) [reply packages/react-docsite-components/src/components/PlatformPicker/PlatformPicker.tsx:36]:
```suggestion
                      iconName: platform.name !== 'Web Components' ? icon : '', // Hide icon to help label fit
```

**Review by ecraig12345 — APPROVED** [review]

**Pull request merged**